bpfmt/bpfmt.go

```diff
@@ -36,8 +37,13 @@ func report(err error) {
 }
 
 func usage() {
+	usageViolation("")
+}
+
+func usageViolation(violation string) {
 	fmt.Fprintf(os.Stderr, "usage: bpfmt [flags] [path ...]\n")
 	flag.PrintDefaults()
+	fmt.Fprintln(os.Stderr, violation)
```

Comment:
I think this traditionally is printed first before usage:

Reply:
done